minio/api.py

```diff
@@ -2602,7 +2604,7 @@ def delete_object_lock_config(self, bucket_name):
             client.delete_object_lock_config("my-bucket")
         """
         self.set_object_lock_config(
-            self, bucket_name, ObjectLockConfig(None, None, None),
+            bucket_name, ObjectLockConfig(None, None, None),
```

Comment:
Now you could keep it one liner if possible.

Reply:
changed